extensions/vscode/src/views/homeView.ts

```diff
@@ -460,22 +454,25 @@ export class HomeViewProvider implements WebviewViewProvider, Disposable {
     if (!savedState) {
       return undefined;
     }
-    return this.getContentRecordByName(
-      savedState.deploymentName,
-      savedState.projectDir,
-    );
+    return this.getContentRecordBySelector(savedState);
   }
 
-  private getContentRecordByName(name: string, projectDir: string) {
+  private getContentRecordBySelector(selector: DeploymentSelector) {
     return this.contentRecords.find(
-      (d) => d.deploymentName === name && d.projectDir === projectDir,
+      (d) => d.deploymentPath === selector.deploymentPath,
```

Comment:
This becomes a unique match, due to the deploymentPath containing the full system path.